rapid7/README.md

```diff
@@ -0,0 +1,17 @@
+This script gets Rapid7 report based on report id and instance provided.
+Parses report and pushes device vulnerabilities to WootCloud custom device context API.
+
+## Steps to run script:
+- Install requirements `pip install -r requirements.txt`
+- Run script `python3 custom_device_context.py  
+  --client_id <WootCloud API client id> 
+  --secret_key <WootCloud API secret key> 
+  --username <Rapid7 username> 
+  --pwd <Rapid7 password> 
+  --id <Rapid7 report ID> 
```

Comment:
change to `--report-id`

Reply:
There is already a report id in script which is the report name/id in Vuln

Reply:
I am just referring to the command line option. Since `--id` is too generic.